### PR TITLE
BG46442 - Add polygon deployment

### PR DESCRIPTION
**Pull request by mithungudipati**

### CODE REVIEW

File: .github/workflows/deploy_and_release.yml [hunk 1]
```diff
@@ -31,11 +99,13 @@ jobs:
           node-version: 14.x
           cache: 'npm'
       - run: npm install
-      - run: npm run deploy-prod
+      - run: npm run deploy-test
```
[the remark this answers is not in the document]
Comment (karankurbur) [reply .github/workflows/deploy_and_release.yml:102]: and its missing the network parameter

[the remark this answers is not in the document]
Comment (mithungudipati) [reply .github/workflows/deploy_and_release.yml]: fixed the  workflow step.

File: hardhat.config.ts [hunk 2]
```diff
@@ -42,11 +48,12 @@ const config: HardhatUserConfig = {
       accounts: [`${PRIVATE_KEY}`]
     },
     matic: {
-      url: `https://polygon-mainnet.g.alchemyapi.io/v2/${ALCHEMY_API_KEY}`,
+      url: `https://polygon-mainnet.g.alchemyapi.io/v2/${POLYGON_API_KEY}`,
```
[the remark this answers is not in the document]
Comment (mithungudipati) [reply hardhat.config.ts:51]: whats the proposed env variable name?

**Review by karankurbur — APPROVED** [review]

File: hardhat.config.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (karankurbur) [reply hardhat.config.ts:51]: ALCHEMY_POLYGON_API_KEY
ETHERSCAN_POLYGON_API_KEY

format should be: SERVICE_NETWORK_API_KEY

[the remark this answers is not in the document]
Comment (karankurbur) [reply hardhat.config.ts:51]: if network doesnt apply, just skip it (for etherscan)

**Pull request merged**